### ISSUE TITLE
Darkjet

### ISSUE THREAD

**Issue by bastorer** [matplotlib/matplotlib#4973]
Hello,

We'd like to submit a proposed colour map for matplotlib 2.0 (or later). Darkjet is a modified version of jet that darkens the green region to improve the appearance.

Regards,
Ben

``` code
>
> import matplotlib
> import matplotlib.cm as colmap
> import matplotlib.colors as mplc
> import matplotlib.pyplot as plt
> import numpy as np
>
> \#\# Through a jet, darkly.
>
> \#\# Create DarkJet
> \#\# ------
> col_list = colmap.jet(range(256))
> darken = np.tile((1 - 0.5/np.cosh(np.linspace(-10,10,256))).reshape((256,1)), [1,col_list.shape[1]])
>
> col_list = col_list*darken # Apply the darkening
>
> \# Matplotlib defaults to rgba, not rgb. 
> \# Reset the transparency values now.
> col_list[:,3] = 1.
>
> darkjet = mplc.ListedColormap(col_list, 'darkjet', 256)
> plt.register_cmap(cmap=darkjet)
> \#\# ------

```

**Comment by tacaswell** [comment]
Strongly :-1: This does not address any of the fundamental problems with jet.

code cleaned up so it is copy-pasteable:

```python
import matplotlib
import matplotlib.cm as colmap
import matplotlib.colors as mplc
import matplotlib.pyplot as plt
import numpy as np

#\# Through a jet, darkly.

#\# Create DarkJet
#\# ------
col_list = colmap.jet(range(256))
darken = np.tile((1 - 0.5/np.cosh(np.linspace(-10,10,256))).reshape((256,1)), [1,col_list.shape[1]])

col_list = col_list*darken # Apply the darkening

# Matplotlib defaults to rgba, not rgb. 
# Reset the transparency values now.
col_list[:,3] = 1.

darkjet = mplc.ListedColormap(col_list, 'darkjet', 256)
plt.register_cmap(cmap=darkjet)
```

![so](https://cloud.githubusercontent.com/assets/199813/9399839/f12b6b62-4787-11e5-9d59-e58a71dd69a3.png)

**Comment by WeatherGod** [comment]
but.. but... it is so colorful! shiny!

/me goes puts a dollar in the jet jar...

Actually, I have to say, it is very colorblind-unfriendly. You have dark at
the top bottom and middle. And the perceptual deltas is all over the place,
leading one to believe there are strong gradients where there aren't. I am
also -1 on this as well.

Sorry.

On Thu, Aug 20, 2015 at 10:15 PM, Thomas A Caswell <notifications@github.com

**Comment by bastorer** [comment]
Thank you for the feedback, those are valid concerns! Can you recommend a more optimal colour map that I could use, or will the new default map work?

**Comment by tacaswell** [comment]
I strongly suggest you watch https://youtu.be/xAoljeRJ3lU 

The four new color maps are currently on master and will be included in 1.5 (which should be out 'soon').

**Comment by tacaswell** [comment]
Closing, if anyone feels strongly the other way please ping to have this reopened.
